integrations/http_bin_org.go

```diff
@@ -0,0 +1,65 @@
+package integrations
+
+import (
+	"github.com/buger/jsonparser"
+	"io/ioutil"
+	"net"
+	"net/http"
+)
+
+const HttpBinExtIPEndpoint = "http://httpbin.org/ip"
+
+type HttpBin struct {
+	Endpoint string
+	Resolved net.IP
+}
+
+var httpBinApi *HttpBin
+
+//InitHttpBin
+//function to initialize httpBinApi
+func InitHttpBin() (err error) {
```

Comment:
Codacy found an issue: [func InitHttpBin should be InitHTTPBin](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)